pkg/monitoring/rules/alerts/prometheus.go

```diff
@@ -0,0 +1,55 @@
+package alerts
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/machadovilaca/operator-observability/pkg/operatorrules"
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+)
+
+const (
+	prometheusRunbookAnnotationKey = "runbook_url"
+	partOfAlertLabelKey            = "kubernetes_operator_part_of"
+	partOfAlertLabelValue          = "kubevirt"
+	componentAlertLabelKey         = "kubernetes_operator_component"
+	componentAlertLabelValue       = "ssp-operator"
+	defaultRunbookURLTemplate      = "https://kubevirt.io/monitoring/runbooks/%s"
+	runbookURLTemplateEnv          = "RUNBOOK_URL_TEMPLATE"
+)
+
+func Register() error {
+	alerts := [][]promv1.Rule{
```

Comment:
This slice of slices variable is not needed, because it always has only one element. You can use the `operatorAlerts()` slice directly.